### ISSUE TITLE
Switch entity not working

### ISSUE THREAD

**Issue by Daandeve** [yangqian/hass-cozylife#1]
I can't get the switch entity to work, error below.
```
Error while setting up cozylife platform for switch
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 249, in _async_setup_platform
    await asyncio.shield(task)
  File "/usr/local/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/cozylife/switch.py", line 39, in setup_platform
    _LOGGER.info(f'ip={hass.data[DOMAIN]}')
KeyError: 'cozylife'
```

**Comment by yangqian** [comment]
I will be receiving some switches soon and will test them.

**Comment by Daandeve** [comment]
Okay, when I remove this line  `_LOGGER.info(f'ip={hass.data[DOMAIN]}')` the error is gone but the switch does not show.
How can I debug this, `custom_components.cozylife: debug` does not work

**Comment by yangqian** [comment]
> 

I have no idea of my recent commits work or not. If you want, you can test it out now. You have to change the "switchs" to "switches" in the config file.

**Comment by Daandeve** [comment]
I just fixed the problem the switches are working now, I'll create a pull request

**Comment by Daandeve** [comment]
Pull request opened #2

**Comment by Daandeve** [comment]
Merged in b5eb0d8c985e001ce6853d04438e34304e24aba7
